.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [
+      1,
+      {
+        "js": "always",
+        "json": "always"
+      }
+    ]
+  },
+  "ignorePatterns": ["dist/", "build/"]
```

Comment:
- [ ] You have done great work so far by installing eslint. However, as the previous reviewer suggested, please make sure that the eslint passes on the pull request.  Ensuring that the eslint is working, it helps you find and fix problems with your JavaScript code.

- Eslint missing 😎 

![linters](https://user-images.githubusercontent.com/99394374/215703145-156d8d4f-9373-4856-90ee-68f7a62f0c2c.png)

Reply:
Thanks for the review @AlphaNtihinduka